src/domains/gps-app/00_alert_gpd_payments_pull.tf

```
@@ -0,0 +1,69 @@
+resource "azurerm_monitor_scheduled_query_rules_alert" "opex_pagopa-gpd-payments-pull-rest-responsetime-upd" {
+  count               = var.env_short == "p" ? 1 : 0
```

I suggest to switch off the _response time alert_.

---

create a var `flag_responsetime_alert` with value `0` and use like this:
```suggestion
  count               = var.flag_responsetime_alert 
```